lib/cmdline.c

```diff
@@ -1435,24 +1346,30 @@ bool rm_cmd_parse_args(int argc, char **argv, RmSession *session) {
         {NULL                     , 0   , HIDDEN           , 0                     , NULL                         , NULL                                                          , NULL}
     };
 
+    const GOptionEntry deprecated_option_entries[] = {
```

Comment:
I'd envisaged overlapping them for at least one release to give time for existing users and external documentation (https://btrfs.wiki.kernel.org/index.php/Deduplication) to move across to the new syntax